tests/fixtures/crypto_top_of_book.yaml

```diff
@@ -0,0 +1,24 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: ['gzip, deflate']
+      Authorization: [Token 0000000000000000000000000000000000000000]
+      Connection: [keep-alive]
+      Content-Type: [application/json]
+      User-Agent: [tiingo-python-client 0.3.2]
+    method: GET
+    uri: https://api.tiingo.com/tiingo/crypto/top/?tickers=btcusd,fldcbtc&includeRawExchangeData=true
```

Comment:
This is subtle, but I think all the generated URIs probably don't have a `/` at the end, were these URLs from an earlier version of the code client? In other words, based on the code I would have expected this to look like

```
https://api.tiingo.com/tiingo/crypto/top?tickers=btcusd,fldcbtc&includeRawExchangeData=true
```